From 4c4895326facbefd6e6417b5a12084e8ff9814bf Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Thu, 16 Mar 2023 17:24:46 +0530
Subject: [PATCH 061/500] OpenWrt: 655-increase_skb_pad.patch

kernel: add a few patches for avoiding unnecessary skb reallocations - significantly improves ethernet<->wireless performance

lede-commit: 6f89cffc9add6939d44a6b54cf9a5e77849aa7fd
Signed-off-by: Felix Fietkau <nbd@nbd.name>

Change-Id: Iffb405d19618f415433537263ed8276ff0f422a0
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
(cherry picked from commit ee0fae0fdbfe9b4f93b5a1f50dde1b5e1f48f123)
---
 include/linux/skbuff.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index 97bfef071255..d209a02b415d 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -3041,7 +3041,7 @@ static inline int pskb_network_may_pull(struct sk_buff *skb, unsigned int len)
  * NET_IP_ALIGN(2) + ethernet_header(14) + IP_header(20/40) + ports(8)
  */
 #ifndef NET_SKB_PAD
-#define NET_SKB_PAD	max(32, L1_CACHE_BYTES)
+#define NET_SKB_PAD	max(64, L1_CACHE_BYTES)
 #endif
 
 int ___pskb_trim(struct sk_buff *skb, unsigned int len);
-- 
2.34.1

